internal/pkg/cli/env_deploy.go

```diff
@@ -126,6 +126,10 @@ func (o *deployEnvOpts) Ask() error {
 	return o.validateOrAskEnvName()
 }
 
+func (o *deployEnvOpts) isManagedCDNEnabled(mft *manifest.Environment) bool {
+	return mft.CDNConfig.CDNEnabled() && mft.HTTPConfig.Public.Certificates == nil && o.targetApp.Domain != ""
```

Comment:
nit: i feel like the public method should just be `Enabled()` since it belongs to `CDNConfig`

Reply:
This causes a conflict with the variable `Enabled *bool`, not very big of an issue, but I need to think of what to rename that variable to for this to work first.

Reply:
One option is also to move the method to the `mft` level :) The pattern that @paragbhingre  has been following is like this:
```go
func (cfg *environmentVPCConfig) CDN() (AdvancedCDNConfig, bool) {
	if cfg.CDNConfig.isEmpty() {
		return nil, false
	}
	return cfg.CDNConfig.Advanced, true
}
```
the clients can then do:
```go
_, cdnEnabled := mft.CDN()
```